app/models/storage_item.rb

```diff
@@ -0,0 +1,3 @@
+class StorageItem < ApplicationRecord
+    belongs_to :places
```

Comment:
This would probably be `belongs_to :place` (i.e. the storage item only belongs to one place)